service/frontend/templates/ratelimited.tmpl

```diff
@@ -45,6 +45,9 @@ import (
 {{$ratelimitTypeMap = set $ratelimitTypeMap "ListWorkflowExecutions" "ratelimitTypeVisibility"}}
 {{$ratelimitTypeMap = set $ratelimitTypeMap "ScanWorkflowExecutions" "ratelimitTypeVisibility"}}
 
+{{$ratelimitTypeMap = set $ratelimitTypeMap "StartWorkflowExecutionAsync" "ratelimitTypeAsync"}}
+{{$ratelimitTypeMap = set $ratelimitTypeMap "SignalWithStartWorkflowExecutionAsync" "ratelimitTypeAsync"}}
```

Comment:
is the build or test going to fail when someone adds a new endpoint but misses to specify it here?

Reply:
unfortunately, it doesn't. Because we have some endpoint with no limit.

Reply:
if we add a no-op ratelimiter and associate all those endpoints with it then we can add a check for existence of endpoint in this template. more future proof then current way. we don't have a runbook for "Add new endpoint" so it's easy to miss